### PR TITLE
docs: add lb4, processmaker3, mixer provider links

### PR DESCRIPTION
**Pull request by ckoliber**
I've created some provider plugins, this pull request will add their links:

1. **Loopback4 CRUD**: a provider fully compatible with [Loopback4 CRUD](https://github.com/loopback4/loopback-component-crud) component, supports `nested create`, `nested update`, `authentication`, `authorization`
2. **ProcessMaker3**: a provider for ProcessMaker3 BPMS
3. **Mixer**: a plugin for combining and merging other providers

### CODE REVIEW

**Review by djhi — CHANGES_REQUESTED** [review]
Thanks! Would mind adding them in alphabetical order?

**Review by ckoliber — COMMENTED** [review]
docs: change DataProviders links order in alphabetical order

**Comment by fzaninotto** [comment]
What's the difference between your loopback adapter and the one already referenced?

**Comment by ckoliber** [comment]
My component, is fully compatible with my loopback [CRUD component](https://github.com/loopback4/loopback-component-crud),
it supports some advanced features:

1. Nested create
2. Nested update
3. [Aggregation queries](https://github.com/loopback4/ra-data-lb4#model-aggregation)

Also it supports typings

File: docs/DataProviders.md [hunk 1]
```diff
@@ -87,13 +87,16 @@ Developers from the react-admin community have open-sourced Data Providers for m
 * **[LocalStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)**: [marmelab/ra-data-localstorage](https://github.com/marmelab/react-admin/tree/master/packages/ra-data-localstorage)
 * **[Loopback3](https://loopback.io/lb3)**: [darthwesker/react-admin-loopback](https://github.com/darthwesker/react-admin-loopback)
 * **[Loopback4](https://loopback.io/)**: [elmaistrenko/react-admin-lb4](https://github.com/elmaistrenko/react-admin-lb4)
+* **[Loopback4 CRUD](https://loopback.io/)**: [loopback4/ra-data-lb4](https://github.com/loopback4/ra-data-lb4)
```
Comment (fzaninotto) [inline docs/DataProviders.md:90]: you should change the initial link to point to the loopback-crud-component package instead

**Review by fzaninotto — CHANGES_REQUESTED** [review]

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**